workspaces/jenkins/plugins/jenkins-backend/src/service/jenkinsInfoProvider.ts

```diff
@@ -50,6 +50,7 @@ export interface JenkinsInfo {
   baseUrl: string;
   headers?: Record<string, string | string[]>;
   jobFullName: string; // TODO: make this an array
+  projectCountLimit?: number;
```

Comment:
This should be added here: https://github.com/backstage/community-plugins/blob/3ce05780029b054117268e6ebf5b54302bbdcc43/workspaces/jenkins/plugins/jenkins-backend/config.d.ts#L18

Reply:
Actually this one doesn't look to be used just the one from the instances, can you clarify this, please?

Reply:
I guess its necessary due to the return value of `getInstanceConfig()`, which returns an `JenkinsInstanceConfig`

https://github.com/backstage/community-plugins/blob/3ce05780029b054117268e6ebf5b54302bbdcc43/workspaces/jenkins/plugins/jenkins-backend/src/service/jenkinsInfoProvider.ts#L258